keras_hub/src/utils/transformers/export_gemma_to_safetensor.py

```diff
@@ -0,0 +1,161 @@
+import json
```

Comment:
lets make this a model-agnostic export utility. Rename file to safetensor_exporter.py
add a dict to maintain the mapping
```
MODEL_EXPORTERS = {
                   "GemmaBackbone": gemma_exporter.get_gemma_weights_map,
                   "LlamaBackbone": llama_exporter.get_llama_weights_map, # Future
}
```
and a user facing API function for the export 
```
def export_to_safetensors(keras_model):
...
```

Reply:
Implement exporter mapping for each model - for this PR's scope just the Gemma model that can serve as a prototype for other models

Reply:
Yeah, let's land this PR first and do this in a separate PR: https://github.com/keras-team/keras-hub/pull/2290#discussion_r2157336378